sweep.py

```diff
@@ -87,8 +87,9 @@ def Bs(ix):
         expr_km[i*3+2,j*3+2] = Maa
 
 #Element force vector
-for i in range(0,4):
-    expr_f[i*3] = q*integrate(integrate(NN[i]*NN[j],(xi,-1,1)),(ny,-1,1))
+for j in range(0,4):
```

Comment:
Added j loop on outer nest.
But I think it has worked without trouble when it was not modified.